meson.build

```diff
@@ -0,0 +1,845 @@
+project('eigen3', 'cpp', version: '3.3.4', lincese: 'MPL2')
+
+eigen_all_files = files(
```

Comment:
Do you actually use `eigen_all_files` somewhere?

Reply:
No. However it's there because some files (like the unsupported bits) may move into use eventually. Also it means it will be obvious if the library adds new files. we can remove these if you like.